### DISCUSSION TITLE
GitHub Discussion or Discord?

### DISCUSSION THREAD

**Discussion by windingwind** [windingwind/zotero-better-notes#1022] [Polls]
It's been a while since I started using GitHub Discussions. Are you satisfied with it?

It seems like the discussion isn't very active, and most of the time, I'm the only one answering questions. Not many users seem to get notified about new posts in the discussion. 

So, do we need a forum on a better platform, for example, Discord?

**Comment by ulahcherubim** [comment]
As we can see from the results of a 2 week old poll having only 2 votes (I doubt it is only yours and mine), I think a migration is necessary. But the ability to store note templates in an easily accessible format is also helpful on github. And there is the risk of people not reading previous discussions before asking the same questions over and over again. 
Maybe the best way to proceed is keeping template sharing and bug reporting exclusively on github and limiting discord channel's scope to questions, ideas and basic help. I think template help should be redirected to github discussions because it is way better to store information about code snippets in github.

**Reply by windingwind** [reply]
Thanks. Bug reports are always through issues on GitHub. The biggest problem with GitHub discussion is that people never get notified of new posts.

**Comment by llity** [comment]
telegram 怎么样？Discord是电子开发的，占用高。

**Reply by windingwind** [reply]
即时通讯工具不太适合承载论坛吧？

**Reply by llity** [reply]
Discord用过，使用起来不太友好。telegram讨论方便，也有channel，也能分组，挺方便的
<img width="935" alt="image" src="https://github.com/user-attachments/assets/49f42d1c-f957-4987-a659-d652a5687be1">
<img width="361" alt="image" src="https://github.com/user-attachments/assets/8d3b59b9-af57-4c90-b7e3-0620009fdcbf">

**Reply by llity** [reply]
使用起来其实蛮好的，希望开一个。
<img width="538" alt="image" src="https://github.com/user-attachments/assets/5f585a55-11e0-446b-a215-4330b9878cd3">

**Reply by windingwind** [reply]
主要我可能没有太多时间回复消息，论坛的话相对即时性要求低一些。或者就是，仍然沿用这个discussion作论坛，即时通讯作为补充

**Reply by llity** [reply]
大家一起回答问题，关键是有活跃度、关注度。才会去回答。GitHub确实没及时性，Discord用过感觉不太友好，使用上有一定限制，TG活跃度肯定高很多。区别 https://icoda.io/zh-hans/telegram-vs-discord-what-are-they/

**Reply by windingwind** [reply]
创建了一个，https://t.me/bnzotero

但是我自己时间精力不多，最好是有热心朋友可以当管理员，经常活跃管理一下。我可能主要就一些讨论性的内容请大家参考，或发布一些版本更新信息（可以考虑用个bot把更新日志推送过去）

**Reply by llity** [reply]
支持啊

**Reply by llity** [reply]
频道可以用于版本发布讨论，群组用于互动交流。

**Reply by windingwind** [reply]
谢谢，我在频道里分享了群组链接，欢迎进群组来讨论

**Comment by windingwind** [comment]
Thank you for all the comments. I have set up a Telegram channel for sharing news and updates about this project. We may also create chat groups for different purposes in the future.

As I may not have much time to manage the channel, if anyone is willing to help with administering the channel, please let me know and I can assign you as an administrator.

The channel is https://t.me/bnzotero

Thank you all!

**Comment by windingwind** [comment]
Hi everyone,

I've linked the [TG group](https://t.me/boost/bnzotero) to the homepage and did some setups. Now you can get notified of any new releases!

I am considering recruiting some volunteers to maintain the doc from the group later. Let's see if people are interested in the TG group.

Thank you all for the discussion!
